Sources/SwiftDocC/Infrastructure/Workspace/DefaultAvailability.swift

```diff
@@ -36,32 +44,82 @@ public struct DefaultAvailability: Codable, Equatable {
         enum CodingKeys: String, CodingKey {
             case platformName = "name"
             case platformVersion = "version"
+            case unavailable = "unavailable"
+        }
+        
+        /// The diferent availability states that can be declared.
+        /// Unavailable or Available with an introduced version.
+        public enum State: Equatable, Hashable, Encodable {
+            case unavailable
+            case available(version: String)
         }
 
         /// The name of the platform, e.g. "macOS".
         public var platformName: PlatformName
+        
+        /// The availability state, e.g unavailable
+        public var state: State
 
         /// A string representation of the version for this platform.
-        public var platformVersion: String
+        @available(*, deprecated, message: "Use `introducedVersion` instead.")
```

Comment:
Please include information about how long this API will be around.

Also, if you provide a `renamed` value then the compiler can create a fixit that updates the call site to the new API.

```suggestion
        @available(*, deprecated, renamed: "introducedVersion", message: "Use 'introducedVersion' instead. This deprecated API will be removed after 5.11 is released")
```

Reply:
✅